index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" href="normalize.css" type="text/css" media="screen">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <title>Jack Wood-Pearce</title>
+</head>
+<body>
+<header id="home">
+    <a href="#home"><h1>Jack Wood-Pearce</h1></a>
+    <nav>
+        <a href="#portfolio"><div class="nav-1">PORTFOLIO</div></a>
```

Comment:
<div> (block-element) in <a> (inline-element) is invalid CSS

Reply:
its a work around to make the whole of the navigation buttons clickable